cookbooks/travis_build_environment/recipes/mysql.rb

```diff
@@ -60,6 +75,16 @@
     > SET PASSWORD FOR 'root'@'localhost' = PASSWORD('');
     > SET PASSWORD FOR 'root'@'127.0.0.1' = PASSWORD('');
   EOF
+  not_if { mysql_version == 5.7 }
+end
+
+file mysql_users_passwords_sql do
+  content <<-EOF.gsub(/^\s+> /, '')
+    > SET old_passwords = 0;
+    > CREATE USER 'travis'@'%' IDENTIFIED BY '';
+    > SET PASSWORD FOR 'root'@'localhost' = PASSWORD('');
+  EOF
+  only_if { mysql_version == 5.7 }
```

Comment:
As per documentation "https://dev.mysql.com/doc/refman/5.7/en/default-privileges.html": "MySQL_ 5.7 create only a 'root'@'localhost' account, but for earlier versions, there might be multiple accounts including 'root'@127.0.0.1". Hence in case of version 5.7 "SET PASSWORD FOR 'root'@'127.0.0.1' = PASSWORD('');" fails and I had to omit the statement.